index.Rmd

```diff
@@ -1,6 +1,8 @@
 ---
 title: animint2 gallery
-output: html_document
+output:
+  html_document:
+    self_contained: false
```

Comment:
this causes index.html to use links to images (28kb), rather than the image data (10MB)